File: sub-model/src/main/java/org/github/base/service/impl/SysOrgServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.SysOrgEntity;
import org.github.base.mapper.ISysOrgMapper;
import org.github.base.service.ISysOrgService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 系统-组织表 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2019-05-07
 */
@Service
public class SysOrgServiceImpl extends ServiceImpl<ISysOrgMapper, SysOrgEntity> implements ISysOrgService {}

File: sub-mysql-sccore-service/src/main/java/org/github/mysql/sccore/base/service/impl/SysOrgServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.sccore.base.service.impl;

import org.github.mysql.sccore.base.entity.SysOrgEntity;
import org.github.mysql.sccore.base.mapper.ISysOrgMapper;
import org.github.mysql.sccore.base.service.ISysOrgService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 区域表，行政数据 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysOrgServiceImpl : ServiceImpl<ISysOrgMapper, SysOrgEntity>(), ISysOrgService
